### PR TITLE
Adding the first draft of the CEE Playlist 

### PR DESCRIPTION
**Pull request by sferich888**
Required gathering items/resources that support will need to start looking at a cluster.

### CODE REVIEW

File: collection-scripts/gather [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Resource List
+resources=()
+
+# Operator Resources
+resources+=(clusteroperators)
+
+# Certificate Resources
+resources+=(csr)
```
Comment (sferich888) [inline collection-scripts/gather]: @deads2k I assume given the threads you have included me in this is a thing we want to collect.

Comment (sferich888) [reply collection-scripts/gather]: I updated this to be plural thus why its out dated.

**Comment by sferich888** [comment]
@deads2k @mfojtik can one of you give this a review (on the direction we want to take this). 
I feel its ok, but don't want to merge my own PR's with our someone first reviewing it. 

cc: @RickJWagner

File: collection-scripts/gather (hunk 1 above)
Comment (sanchezl) [reply collection-scripts/gather]:
```suggestion
resources+=(certificatesigningrequests)
```
`csr` is just a short name, so the plural form is `certificatesigningrequests`:
```console
[sanchezl@monolith must-gather]$  ./openshift-must-gather inspect csrs --base-dir CSRS
Error: the server doesn't have a resource type "csrs"
[sanchezl@monolith must-gather]$  ./openshift-must-gather inspect certificatesigningrequests --base-dir CSR
2019/04/01 21:15:22 Finished successfully with no errors.
[sanchezl@monolith must-gather]$ tree CSR/
CSR/
└── cluster-scoped-resources
    └── certificates.k8s.io
        └── certificatesigningrequests
            ├── csr-29q65.yaml
            ├── csr-49n9f.yaml
            └── csr-9qwh4.yaml

3 directories, 3 files
[sanchezl@monolith must-gather]$ 
```

Comment (sferich888) [reply collection-scripts/gather]: Fixed

File: collection-scripts/gather [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+resources+=(clusteroperators)
+
+# Certificate Resources
+resources+=(certificatesigningrequests)
+
+# Machine/Node Resources
+resources+=(nodes machines machineconfigs)
+
+# Namespaces/Project Resources
+resources+=(ns/default ns/openshift ns/kube-system)
```
Comment (mfojtik) [inline collection-scripts/gather]: why is `ns/openshift` interesting?

**Comment by RickJWagner** [comment]
I've reviewed this PR and find it good.  (I think all of the listed resources have potential value, depending on what needs to be investigated.)
We may want to add to this over time.  (Example:  I've had some recent work that dealt with users/groups/roles, the information wasn't easy to gather.  This might be my own lack of experience, though, so I don't suggest any changes for the time being.)

File: collection-scripts/gather (hunk 2 above)
Comment (sferich888) [reply collection-scripts/gather]: To pull content from the OpenShift Namespace? 
I'm not sure what this question is trying to ask.

**Pull request merged**